devtools/php-cs-fixer/run-php-cs-fixer.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#   BAREOS® - Backup Archiving REcovery Open Sourced
+#
+#   Copyright (C) 2023 Bareos GmbH & Co. KG
+#
+#   This program is Free Software; you can redistribute it and/or
+#   modify it under the terms of version three of the GNU Affero General Public
+#   License as published by the Free Software Foundation and included
+#   in the file LICENSE.
+#
+#   This program is distributed in the hope that it will be useful, but
+#   WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#   Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program; if not, write to the Free Software
+#   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+#   02110-1301, USA.
+
+./vendor/bin/php-cs-fixer fix ../../webui -v \
+    --rules=@PSR12 \
+    --dry-run \
+    --using-cache=no \
+    --stop-on-violation
```

Comment:
```suggestion
DIR="$(dirname $0)"
cd "${DIR}"
./vendor/bin/php-cs-fixer fix ../../webui \
    -v \
    --rules=@PSR12 \
    --stop-on-violation "$@"
```
Make it callable from other directories, and adapt the call in a way it can be used for automatic code formatting (remove --dry-run), and add passing additonal parameter (like --help).